ga4gh/datamodel/variants.py

```diff
@@ -994,7 +994,10 @@ def convertSeqOntology(self, seqOntStr):
             so = self._createGaOntologyTermSo()
             so.term = soName
             if self._sequenceOntology is not None:
-                so.id = self._sequenceOntology.getId(soName)
+                try:
```

Comment:
Is this really the semantics we want (using the empty string when the key isn't defined)? If so, how about we change the ``getId`` function to follow the semantics of ``dict.get``, and add a second parameter to give the default value when the key is not found? So, here we'd have ``so.id = self._sequenceOntology.getId(soName, "")``